SQL/paradise_schema.sql

```diff
@@ -446,10 +446,13 @@ DROP TABLE IF EXISTS `erro_watch`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `erro_watch` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
   `ckey` varchar(32) NOT NULL,
   `reason` text NOT NULL,
-  PRIMARY KEY (`id`)
+  `timestamp` datetime NOT NULL,
+  `adminckey` varchar(32) NOT NULL,
```

Comment:
Their PR isn't even 12 hours old yet. The ID column is specifically for use with the index, and it would be highly inconsistant for both our database and their database to base the index off of the ckey instead.